### PR TITLE
Major refactoring including Swagger UI update.

### PR DESCRIPTION
**Pull request by garster**
- Remove included swagger, replace with CDN.
- Reorganize files
- Cleanup gitignore, gitattributes, etc.

Running at https://drupal7.garyscott.net/node/50

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Create a new Content Type
+
+
+- Go to admin/structure/types (Add Content Type)
+- Create Content Type with the name of your choosing. Click Save and add fields.
+- Delete Body field, add File field with a name of your choosing. Click Save.
+- On the Edit tab set allowed file extensions to `json, yaml, yml`.
+- Set a File directory of your choosing. Save settings.
+- On the Manage display tab, for the file field set Label to `<Hidden>` and the Format to `Swagger UI Formatter`. Click Save.
+- Click gear button on the right. Good options are Doc Expansion=List, and Show Request Headers=checked. Click Update.
```
Comment (dsaludares) [inline README.md]: Add "Click Save" after "Click Update." Otherwise, your changes on this Manage display tab will not be saved.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+This module provides a field formatter for file fields with allowed file types
+as json or yml, which renders the uploaded json or yml file using the Swagger UI
+if the uploaded json or yml file is a valid swagger file. This module uses the
+Swagger UI Library available at https://github.com/swagger-api/swagger-ui
+
+## Requirements
+
+This module does not have any special requirements. However in order to be able
+to allow json or yml file extensions in Drupal, you might need to install the
+file entity module.
```
Comment (dsaludares) [inline README.md]: Just as an FYI, there is an update to the File Entity (fieldable files). The dev.developer site is running 7.x-2.12, but the recommended version is 7.x-2.16.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Swagger UI Field Formatter
+
+## Introduction
+
+This module provides a field formatter for file fields with allowed file types
+as json or yml, which renders the uploaded json or yml file using the Swagger UI
```
Comment (dsaludares) [inline README.md]: Do we want to add "yaml" as well, or is it just understood that "yaml" is included? "yaml" is mentioned in line 30 below.

**Comment by garster** [comment]
A good future enhancement would be to make a custom layout so we can remove the title if wanted. We should also look into rewriting all the code so we can remove ourselves from the original author.

**Comment by dsaludares** [comment]
A few more comments:

- I was wondering if it is your intention to publish the location of the actual file, e.g., https://drupal7.garyscott.net/sites/default/files/openapi/cams-v1.yaml
- The version number looks like superscript. Is this intended?
- "Authorize" is not centered on the "Authorize" button.
- Do we want to try to change the colors to use the UCSB color palette or leave as is?

Thanks again, @garster ! You do amazing work!

**Comment by garster** [comment]
The UI is out-of-the-box and there might be some styling conflicts that make that version look weird. See http://petstore.swagger.io/ for the default layout.

**Pull request merged**